cypress/support/commands.ts

```diff
@@ -0,0 +1,37 @@
+/// <reference types="cypress" />
```

Comment:
Why is this file commented out? is it needed?

Reply:
This file is automatically generated when you initialize Cypress. It's not being used now but i left it since it might be useful in the future.